content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -32,175 +32,192 @@ To create a new kata, you need to earn the ["Create Kata" privilege](/gamificati
+After you publish your kata, it becomes subject to the [beta evaluation](/concepts/kata/beta-process/) process. At this point, power users will immediately jump on it, knowing they _will_ find problems, the discourse section will be flooded with issue reports and, unless you managed to avoid any serious issues, the satisfaction rating might start to drop. This may be discouraging at first, especially for first-time authors, but it is also to be expected; after all, no one wants an unfinished kata to get out of beta, right? That's why it is so important to make sure that when your kata is published, it's already in as good a shape as possible.
 
-But still, no matter how hard you try, it's not possible to make everyone satisfied. Your kata may receive many satisfied votes, but even then, some users might not like it. Do not worry about this too much. Consider their remarks, and think a bit about them. Maybe they are right, and the kata needs some improvements? Listen to everyone, answer their questions, consider suggestions, and fix issues popping up. Also, do not rush things too much. If someone says something you are not sure about, just wait for another opinion. Users often do not agree with each other, so you can receive two mutually exclusive opinions. It can be difficult sometimes, but you just need to pass on one of them, or find some balance in between. Research shows \[citation needed] that kata with authors actively maintaining them leave beta quite quickly, while some kata can be retired within minutes, or, even worse, stay in beta forever, if authors do not want to fix problems.
+That said, no matter how hard you try, you can never satisfy everone. Even if your kata receives overwhelmingly positive votes, some users might still not like it. Do not worry about this too much, but do not disregard their feedback either. Give their remarks some consideration, they may point out ways in which your kata can be improved. Listen to everyone, answer their questions, consider suggestions, and fix any issues that crop up. Also, do not rush things too much. If someone says something you are not sure about, just wait for another opinion. Then again, users tend to disagree a lot, and so might offer contradicting suggestions. It can be difficult, but in such cases you need to either find a compromise or just pick one option over the other and move forward. Your mindset and approach to the beta process matter: experience shows that kata whose authors engage actively and constructively in the process tend to graduate out of beta quite quickly, whereas other kata get retired within minutes, or, perhaps worse, never leave beta status due to the author's unwillingness to accept advice or fix issues.
```

Comment:
```suggestion
That said, no matter how hard you try, you can never satisfy everyone. Even if your kata receives overwhelmingly positive votes, some users might still not like it. Do not worry about this too much, but do not disregard their feedback either. Give their remarks some consideration, they may point out ways in which your kata can be improved. Listen to everyone, answer their questions, consider suggestions, and fix any issues that crop up. Also, do not rush things too much. If someone says something you are not sure about, just wait for another opinion. Then again, users tend to disagree a lot, and so might offer contradicting suggestions. It can be difficult, but in such cases you need to either find a compromise or just pick one option over the other and move forward. Your mindset and approach to the beta process matter: experience shows that kata whose authors engage actively and constructively in the process tend to graduate out of beta quite quickly, whereas other kata get retired within minutes, or, perhaps worse, never leave beta status due to the author's unwillingness to accept advice or fix issues.
```

Reply:
Not sure if I'm missing something in this suggestion as it appears to produce no changes?

Reply:
Hmm, not sure why the diff isn't highlighting `y`. But it's a single character change in a really long line.

Just `s/everone/everyone/` in the first sentence.

Reply:
I was planning to commit the typos myself, but forgot to.